examples/full-example/isomorphic/base/components/toolbar.js

```diff
@@ -1,14 +1,39 @@
 import React from 'react';
 import Button from './button';
 
+let Lazy;
+
 class Toolbar extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            laoded: false
+        }
+        this.handleClick = this.handleClick.bind(this)
+    }
+
+    handleClick() {
+        if (!this.state.loaded) {
+            Lazy = require('./lazy').default;
+
+            this.setState({
+                loaded:true
+            });
+        }
+    }
+
     render() {
         return (
-            <nav className="toolbar">
-                <Button />
-                <Button />
-                <Button />
-            </nav>
+            <div>
+                <nav className="toolbar">
+                    <Button onClick={this.handleClick}>Button #1</Button>
```

Comment:
Line 12 looks ugly, but since we don't use classes I am not sure if this is a good or a bad idea. It actually does the job well, so I think this is ok for now.